### PR TITLE
doc: update test.md about how to run test

### LINKED ISSUE #1320: [test] execute `go test go test -check.f PouchNetworkSuite.TestNetworkConnect` fails
### Ⅰ. Issue Description

When I read the testing document in https://github.com/alibaba/pouch/blob/master/docs/test/test.md, I know how to test integration test.

While when I execute `go test go test -check.f PouchNetworkSuite.TestNetworkConnect`, it fails with the error message:
```shell
root@ubuntu:~/go/src/github.com/alibaba/pouch/test# go test -check.f PouchNetworkSuite.TestNetworkConnect
# github.com/alibaba/pouch/test
cli_network_test.go:15:2: cannot find package "github.com/vishvananda/netlink" in any of:
	/root/go/src/github.com/alibaba/pouch/vendor/github.com/vishvananda/netlink (vendor tree)
	/usr/local/go/src/github.com/vishvananda/netlink (from $GOROOT)
	/root/go/src/github.com/vishvananda/netlink (from $GOPATH)
FAIL	github.com/alibaba/pouch/test [setup failed]
```

I am afraid the guidance in the document does not cover every case, since for some cases GOPATH needs to be set more paths.

### Ⅱ. Describe what happened

CI fails to execute since GOPATH issue.

### Ⅲ. Describe what you expected to happen

Test Document could explicitly cover all the cases to guide users.

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

none

### Ⅵ. Environment:

- pouch version (use `pouch version`):
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

### PR DESCRIPTION
**Pull request by Letty5411**
Signed-off-by: letty <letty.ll@alibaba-inc.com>

### Ⅰ. Describe what this PR did
Update test.md to make sure steps about how to run test are right.
Mainly add steps about setting GOPTAH to find libnetwork package.

### Ⅱ. Does this pull request fix one issue?

fix #1320

### CODE REVIEW

**Comment by allencloud** [comment]
CI fails:

```
----------------------------------------------------------------------
FAIL: /go/src/github.com/alibaba/pouch/test/cli_ps_test.go:113: PouchPsSuite.TestPsNoTrunc
/go/src/github.com/alibaba/pouch/test/cli_ps_test.go:131:
    c.Assert(kv[name].id, check.HasLen, 64)
... obtained string = ""
... n int = 64
```

I am afraid this is a flaky test. Fortunately, @HusterWan already found out a way to figure it out.

**Comment by allencloud** [comment]
Please rebase to the latest master. Thanks a lot. @Letty5411

File: docs/test/test.md [hunk 1]
```diff
@@ -88,13 +88,22 @@ GOROOT=/usr/local/go
...
 
 ```
 # pwd
 /go/src/github.com/alibaba/pouch
 ```
 
+The following steps are also needed to make sure libnetwork package could be found:
+
+```
+BUILDPATH=/tmp/pouchbuild
```
Comment (fuweid) [inline docs/test/test.md]: Please add the remainder: the system will clean all the file or dir in `/tmp` after restart.

**Comment by fuweid** [comment]
LGTM.

Since the fail case doesn't related to this change, I will mere this PR.

**Pull request merged**